qutebrowser/completion/completionwidget.py

```diff
@@ -28,6 +28,7 @@
 
 from qutebrowser.config import config, style
 from qutebrowser.completion import completiondelegate, completer
+from qutebrowser.completion.models.base import BaseCompletionModel
```

Comment:
I usually only import modules and never classes from modules (because that'd get messy fast) - so I'd prefer `from qutebrowser.completion.models import base` and `base.BaseCompletionModel.COLUMN_WIDTHS` here.